### PR TITLE
suppress title for submitted submission

### LINKED ISSUE #1633: Suppress manuscript title on Thank You page and Dashboard for HotJar recordings
As a continuation of #1557 and #1597 we should suppress the title of the manuscript on the Thank You page and also the dashboard for the purposes of HotJar recordings.

screen shot 2019-03-05 at 11 13 30

49295206-2873d700-f4ad-11e8-9fb0-0efcbd2be9a2

tasks: 
- [x] dashboard
- [x] thankyou page

### PR DESCRIPTION
**Pull request by jyenes**
- [x] suppress title from the dashboard
- [x] suppress title from the thank you page

closes: #1633

### CODE REVIEW

**Review by SemraSh — APPROVED** [review]
LGTM 👍

File: test/hotjar.browser.js [hunk 1]
```diff
@@ -104,3 +110,27 @@ test('test disclousure is suppressing name', async t => {
...
+    .ok()
+})
+
+test('checking suppresion on the thankyou page', async t => {
+  const navigationHelper = new NavigationHelper(t)
+  await navigationHelper.paricialSubmissionThankyou(manuscript)
+  await navigationHelper.consentDisclosure()
+  await navigationHelper.submit()
+  await navigationHelper.accept()
+  await t
```
Comment (Cooryd) [inline test/hotjar.browser.js:97]: This is quite a lot of process to check that a data attribute exists on an element on the `ThankYou` page. Could this check not be achieved through a unit test instead?

Comment (Cooryd) [reply test/hotjar.browser.js:97]: And maybe a single browser test for checking suppression? (multiple assertions as it progresses)

**Review by diversemix — APPROVED** [review]

File: test/hotjar.browser.js (hunk 1 above)
Comment (Cooryd) [reply test/hotjar.browser.js:97]: @jyenes I've added a unit test for this to keep the coverage up.

**Review by Cooryd — APPROVED** [review]

File: test/hotjar.browser.js (hunk 1 above)
Comment (jyenes) [reply test/hotjar.browser.js:97]: thanks @Cooryd

**Pull request merged**